src/test/java/com/amazon/dlic/auth/http/jwt/keybyoidc/HTTPJwtKeyByOpenIdConnectAuthenticatorTest.java

```diff
@@ -108,20 +111,94 @@ public void testRoles() throws Exception {
 		Assert.assertEquals(TestJwts.TEST_ROLES, creds.getBackendRoles());
 	}
 
+
 	@Test
 	public void testExp() throws Exception {
 		Settings settings = Settings.builder().put("openid_connect_url", mockIdpServer.getDiscoverUri()).build();
 
 		HTTPJwtKeyByOpenIdConnectAuthenticator jwtAuth = new HTTPJwtKeyByOpenIdConnectAuthenticator(settings, null);
-
+		
 		AuthCredentials creds = jwtAuth.extractCredentials(
 				new FakeRestRequest(ImmutableMap.of("Authorization", TestJwts.MC_COY_EXPIRED_SIGNED_OCT_1),
 						new HashMap<String, String>()),
 				null);
 
 		Assert.assertNull(creds);
+	}	
+
+	@Test
+	public void testExpInSkew() throws Exception {
```

Comment:
Let's clean these tests up by refactoring the comment code.  I'd recommend creating a private function that accepts an expiration time and settings object as parameters with a return of AuthCredentials.

This should alter the test methods to be the setup for the settings, the action to generate the credentials object, and the verification messages.

Reply:
Sorry I don't understand your request, could you explain it ? thanks
Is it OK if I create this function in class TestJwts ?

```
	static String createMcCoySignedOct1(int nbf, int exp)
	{
		JwtToken jwt_token = create(
			MCCOY_SUBJECT, TEST_AUDIENCE, 
			ROLES_CLAIM, TEST_ROLES_STRING, 
			JwtConstants.CLAIM_NOT_BEFORE, nbf, 
			JwtConstants.CLAIM_EXPIRY, exp);

		return createSigned(jwt_token, TestJwk.OCT_1);
	}
```